<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChzzkLink_Alpha</class>
 <widget class="QMainWindow" name="ChzzkLink_Alpha">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>513</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>4000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ChzzkLink_Alpha</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>11</number>
    </property>
    <property name="rightMargin">
     <number>11</number>
    </property>
    <property name="bottomMargin">
     <number>11</number>
    </property>
    <property name="spacing">
     <number>7</number>
    </property>
    <item row="10" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>&lt;a href=&quot;mailto:﻿&quot;root@bums.kr&quot;&gt;Contact: root@bums.kr&lt;/a&gt;</string>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="pixmap">
       <pixmap>icon.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="4">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1" colspan="2">
     <widget class="QPushButton" name="pushButton_nid">
      <property name="text">
       <string>NID입력</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QComboBox" name="comboBox">
      <property name="currentText">
       <string>1080p</string>
      </property>
      <property name="maxCount">
       <number>2147483645</number>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::InsertAfterCurrent</enum>
      </property>
      <item>
       <property name="text">
        <string>1080p</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>720p</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>480p</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>해상도 </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="pushButton_start">
      <property name="text">
       <string>녹화 시작</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>치지직 URL</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="2">
     <widget class="QComboBox" name="comboBox_2">
      <item>
       <property name="text">
        <string>ts</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>mp4</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0" colspan="4">
     <widget class="QLineEdit" name="lineEdit_channel_id"/>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>142</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>142</width>
        <height>80</height>
       </size>
      </property>
      <property name="text">
       <string>          Thumbnail</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>파일 형식</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="pushButton_stop">
      <property name="text">
       <string>녹화 중지</string>
      </property>
     </widget>
    </item>
    <item row="10" column="3">
     <widget class="QPushButton" name="pushButton_end">
      <property name="text">
       <string>종료</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>470</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNID">
   <property name="text">
    <string>NID</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
